api/mutations/mutations.go

```diff
@@ -0,0 +1,71 @@
+package mutations
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+
+	"github.com/dgraph-io/dgo/v240/protos/api"
+	"github.com/dgraph-io/dgraph/v24/protos/pb"
+	"github.com/dgraph-io/dgraph/v24/schema"
+	"github.com/hypermodeinc/modusdb/api/utils"
+)
+
+func HandleReverseEdge(jsonName string, value reflect.Type, nsId uint64, sch *schema.ParsedSchema,
+	jsonToReverseEdgeTags map[string]string) error {
+	if jsonToReverseEdgeTags[jsonName] == "" {
+		return nil
+	}
+
+	if value.Kind() != reflect.Slice || value.Elem().Kind() != reflect.Struct {
+		return fmt.Errorf("reverse edge %s should be a slice of structs", jsonName)
+	}
+
+	reverseEdge := jsonToReverseEdgeTags[jsonName]
+	typeName := strings.Split(reverseEdge, ".")[0]
+	u := &pb.SchemaUpdate{
+		Predicate: utils.AddNamespace(nsId, reverseEdge),
```

Comment:
We should not have a package name `utils`. Each package should have a more specific name based on what it does